string record_name
---
bool succeed
